docs/install-function-mesh.md

```diff
@@ -43,72 +43,78 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
 > - Before installation, ensure that Helm v3 is installed properly.
 > - For the use of `kubectl` commands, see [kubectl command reference](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands).
 
-1. Clone the StreamNative Function Mesh repository.
+1. Add Function Mesh charts repo.
 
     ```shell
-    git clone https://github.com/streamnative/function-mesh.git
-    cd function-mesh
+    helm repo add function-mesh http://charts.functionmesh.io/
+    helm repo update
     ```
 
-2. Create the custom resource type.
+2. Install the Function Mesh Operator.
+
+    Let's set some variables for convenient use later.
```

Comment:
I know setting the environment variable is a convenient way. But maybe it's not suitable for all users.